### PR TITLE
Market Pool tab showed incorrect value for numbers <0.01 on 'Earned', 'Deposited' and 'Total' values

### LINKED ISSUE #1837: Decimal number precision error on "Pool" tab
Earned, Deposited, and Total values aren't formatted correctly

### PR DESCRIPTION
**Pull request by gem1no**
fixes #1837 

small UI fix. added `displayCollateral.decimals` for precision value in `formatBigNumber`

### CODE REVIEW

**Review by hexyls — APPROVED** [review]
Looks good! 

I noticed that there is a similar issue in `UserPoolData`, for both `Your Earnings` and `Total Earnings` if you want to take a look at those too 👍

**Review by KadenZipfel — APPROVED** [review]
lgtm!

**Review by Mi-Lan — CHANGES_REQUESTED** [review]
.

File: app/src/components/market/sections/market_pooling/user_pool_data.tsx [hunk 1]
```diff
@@ -78,8 +78,12 @@ export const UserPoolData: React.FC<Props> = (props: Props) => {
...
     displayTotalEarnings = compoundService.calculateCTokenToBaseExchange(baseCollateral, totalEarnings)
   }
-  const userLiquidityFormatted = formatNumber(formatBigNumber(displayUserLiquidity, baseCollateral.decimals))
-  const poolTokensFormatted = formatNumber(formatBigNumber(displayPoolTokens, baseCollateral.decimals))
+  const userLiquidityFormatted = formatNumber(
+    formatBigNumber(displayUserLiquidity, baseCollateral.decimals, baseCollateral.decimals),
+  )
+  const poolTokensFormatted = formatNumber(
+    formatBigNumber(displayPoolTokens, baseCollateral.decimals, baseCollateral.decimals),
+  )
```
Comment (Mi-Lan) [inline app/src/components/market/sections/market_pooling/user_pool_data.tsx:86]: Works good! I would just move these directly to value field in component property. Just to be consistent with the rest of code.

**Review by Mi-Lan — CHANGES_REQUESTED** [review]
Should be gtg! Just small change

**Review by Mi-Lan — APPROVED** [review]

**Pull request merged**